### PR TITLE
Adding a wedding march song

### PR DESCRIPTION
**Pull request by rexpie**
Not a long version, just the first two bits. Tempo is set to 90 to get into the mood.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "j5-songs",
-  "version": "0.0.7",
+  "version": "0.0.8",
```
Comment (julianduque) [inline package.json]: Please don't update package.json since I do this on a different commit, thanks :)

Comment (rexpie) [inline package.json]: Duely noted.
